src/redist/targets/GenerateLayout.targets

```diff
@@ -47,15 +47,16 @@
       <!-- Note: we use the "-internal" archives and installers that contain only the aspnetcore shared framework, and shouldn't overlap with Microsoft.NETCore.App. -->
       <DownloadedAspNetCoreSharedFxInstallerFileName Condition=" '$(InstallerExtension)' == '.msi' ">aspnetcore-runtime-internal-$(AspNetCoreVersion)-$(AspNetCoreSharedFxInstallerRid).wixlib</DownloadedAspNetCoreSharedFxInstallerFileName>
       <DownloadedAspNetTargetingPackInstallerFileName Condition=" '$(InstallerExtension)' != '' ">aspnetcore-targeting-pack-$(AspNetTargetingPackVersion)-$(AspNetCoreSharedFxInstallerRid)$(InstallerExtension)</DownloadedAspNetTargetingPackInstallerFileName>
-      <!-- The AspNetTargetingPackInstallerFileName is misnamed for DEB types at this time.
-           https://github.com/aspnet/AspNetCore/issues/8835 when resolved, remove the 'IsDebianBaseDistro' special contitioned line below. -->
+      <!-- The AspNet TargetingPack assets are misnamed for DEB and tar.gz types at this time; the RID is missing.
+           https://github.com/aspnet/AspNetCore/issues/8835 -->
```

Comment:
Again, I don't think these need to have RIDs. Misnamed is a bit misleading. I think arch specific targeting packs can be a windows only thing.

Reply:
Agreed - currently, the assets follow a naming convention - these specific assets have deviated from the naming convention. I'll change the wording...